style.css

```diff
@@ -0,0 +1,342 @@
+* {
+	padding: 0;
+	margin: 0;
+}
+
+.wrapper {
+	max-width: 930px;
+	min-width: 320px;
+	font-family: 'GothamPro', serif;
+	margin: 0 auto;
+	box-sizing: border-box;
+}
+
+/* header */
+
+.header {
+	width: 100%;
+	height: 63px;
+	display: grid;
+	flex-direction: row;
+	grid-template-columns: 0.7fr 4fr 1.5fr 0.6fr 0.6fr 0.3fr;
+	margin-bottom: 33px;
+	padding-top: 24px;
+	padding-bottom: 8px;
+}
+
+.header__logo {
+	display: flex;
+	align-items: center;
+}
+
+.header__logo-image {
+	width: 55px;
+	height: 55px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)